### DISCUSSION TITLE
Does It update the Card?

### DISCUSSION THREAD

**Discussion by redshift-cyber** [anuraghazra/github-readme-stats#1282] [Q&A] [answered]
Currently my top langugaes are javaScript,HTML,Python,CSS,EJS and let's say i got into typescript will it update it and add typescript?

**Comment by rickstaa** [comment, marked as answer]
@iron-coder12 The language card will indeed update given that [linguist](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L6386), which we use for our language detection, is aware of typescripts existence. See https://github.com/anuraghazra/github-readme-stats/discussions/809#discussioncomment-279223 for more information.

**Reply by rickstaa** [reply]
@iron-coder12 Did that answer your question?
